client/transactions/list/index.js

```diff
@@ -339,17 +339,25 @@ export const TransactionsList = ( props ) => {
 		);
 	}
 
-	// Generate summary based on loading state and available currencies information
-	const summary = [
-		{
-			label: __( 'transactions', 'woocommerce-payments' ),
-			value: `${ transactionsSummary.count }`,
-		},
-	];
 	const isCurrencyFiltered = 'string' === typeof getQuery().store_currency_is;
-	if ( ! isSummaryLoading ) {
-		const isSingleCurrency =
-			2 > ( transactionsSummary.store_currencies || [] ).length;
+
+	const isSingleCurrency =
+		2 > ( transactionsSummary.store_currencies || [] ).length;
+
+	let summary;
```

Comment:
Similar to comment for deposits listing.

Reply:
explanation same as the comment above ^